BootloaderCommonPkg/Include/Library/ConfigDataLib.h

```diff
@@ -59,11 +59,44 @@ typedef struct {
   //
   UINT8   HeaderLength;
   UINT8   Attribute;
+  union {
+    //
+    // Internal configuration data offset in DWORD from the start of data blob.
+    // This value is only valid in runtime.
+    //
+    UINT16  InternalDataOffset;
+    struct {
+      //
+      // Security version number
+      // This is available only in flash. It would be overwritten by CDATA_BLOB.InternalDataOffset in runtime
+      //
+      UINT8   Svn;
+      // Reserved
+      UINT8   Rsvd;
+    } Addi_Info;

```

Comment:
You can simplify it using:
union {
    //
    // Internal configuration data offset in DWORD from the start of data blob.
    // This value is only valid in runtime.
    //
    UINT16  InternalDataOffset;
    UINT8    Svn;
}  ExtraInfo;

Reply:
Sure. There would be an Reserved byte available which we can add in future.

Reply:
Done.